api-reference/10 UI Components/Errors and Warnings/W1020.md

```diff
@@ -3,7 +3,8 @@ id: ErrorsUIWidgets.W1020
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs if the [shading](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/#shading) property is **false**. In this case, the [hideEvent](/Documentation/ApiReference/UI_Components/dxPopover/Configuration/hideEvent/) property's value is ignored.
```

Comment:
Does it occur any time `shading` is false?

Reply:
No, I was wrong. It occurs any time when it is **true**